.github/workflows/build-docker-artifact.yaml

```
@@ -125,7 +184,7 @@ jobs:
   tag-latest:
     name: "🔄 Update latest tag"
     needs: check-docker-images
-    if: github.event_name == 'push' && github.ref == 'refs/heads/main' && needs.check-docker-images.outputs.ci-build-exists == 'true'
+    if: github.event_name == 'push' && github.ref == 'refs/heads/main' && (needs.check-docker-images.outputs.ci-build-exists != 'true' || needs.check-docker-images.outputs.dev-exists != 'true')
```

We only need to tag `:latest` for dev, so that IRD can pick it up.  This step doesn't care about any other image.

```suggestion
    if: github.event_name == 'push' && github.ref == 'refs/heads/main' &&  needs.check-docker-images.outputs.dev-exists != 'true'
```